core/trino-main/src/main/java/io/trino/operator/scalar/CombineHashFunction.java

```diff
@@ -27,4 +30,13 @@ public static long getHash(@SqlType(StandardTypes.BIGINT) long previousHashValue
     {
         return (31 * previousHashValue + value);
     }
+
+    @UsedByGeneratedCode
+    public static void combineAllHashesWithConstant(long[] hashes, int fromIndex, int toIndex, long value)
+    {
+        int i = checkFromToIndex(fromIndex, toIndex, hashes.length);
+        for (; i < toIndex; i++) {
```

Comment:
Why not just:

```java
checkFromToIndex(fromIndex, toIndex, hashes.length);
for (int i = fromIndex; i < toIndex; i++) {
    ...
}
```

I think that's easier to read, as one doesn't have to look at what `checkFromToIndex` does to understand what's going to be the initial value of `i`.

Reply:
Sure, done.